From 5c701f39f3cb7ec47c0be431648ca4a4440eeedd Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Fri, 2 Mar 2012 16:28:23 +0100
Subject: [PATCH 12/12] Documentation: Mention qcow2 full preallocation

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1330705703-8967-5-git-send-email-kwolf@redhat.com>
Patchwork-id: 37895
O-Subject: [RHEL-6.3 qemu-kvm PATCH 4/4] Documentation: Mention qcow2 full preallocation
Bugzilla: 676484
RH-Acked-by: Ademar de Souza Reis Jr. <areis@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

Bugzilla: 676484
Upstream: n/a (full preallocation doesn't exist upstream)

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
---
 qemu-img.texi |    9 ++++++---
 1 files changed, 6 insertions(+), 3 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-img.texi |    9 ++++++---
 1 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/qemu-img.texi b/qemu-img.texi
index 8fce632..02ffdd0 100644
--- a/qemu-img.texi
+++ b/qemu-img.texi
@@ -214,9 +214,12 @@ sizes can improve the image file size whereas larger cluster sizes generally
 provide better performance.
 
 @item preallocation
-Preallocation mode (allowed values: off, metadata). An image with preallocated
-metadata is initially larger but can improve performance when the image needs
-to grow.
+Preallocation mode (allowed values: off, metadata, full). An image with
+preallocated metadata is initially larger but can improve performance when the
+image needs to grow. Full preallocation additionally writes zeros to the whole
+image in order to preallocate lower layers (e.g. the file system containing the
+image file) as well. Note that full preallocation writes to every byte of the
+virtual disk, so it can take a long time for large images.
 
 @end table
 
-- 
1.7.7.6

